client/blocks/reader-post-card/byline.jsx

```diff
@@ -79,7 +80,7 @@ class PostByline extends React.Component {
 							author={ post.author }
 							siteUrl={ streamUrl }
 							post={ post }>
-							{ post.author.name }
+							{ decodeEntities( post.author.name ) }
```

Comment:
Oddly, this is already in the post normalization rules:

https://github.com/Automattic/wp-calypso/blob/master/client/lib/post-normalizer/rule-decode-entities.js#L27

...and `rule-decode-entities` is in our `fastNormalizationRules` for Reader.

I'll have a play with it and find out what's happening.